BusinessTampereTrafficMonitoring/traffic_lights/api_client.py

```diff
@@ -79,17 +96,19 @@ def store(self, events: List):
                     signal_group=signal_group,
                     t_start=_parse_date(t_start),
                     t_green=_parse_date(t_green),
-                    t_end=_parse_date(t_end)
-                )
+                    t_end=_parse_date(t_end))
                 db_conn.execute(stmt)
-            db_conn.commit()
+                db_conn.commit()
 
     def start_polling(self, interval: float):
         """
         Periodically updates device states and stores events to database.
 
         This method never returns unless another thread calls stop_polling().
         It is intended to be called in a new thread.
+        # Parameter:
+        # argument1: interval (float)
```

Comment:
Suggestion: "The time to wait between polling the API"

Reply:
These comments are already written. Its not from my end.